src/main/java/no/unit/nva/model/instancetypes/PublicationInstance.java

```diff
@@ -8,7 +8,10 @@
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonSubTypes({
         @JsonSubTypes.Type(name = "JournalArticle", value = JournalArticle.class),
-        @JsonSubTypes.Type(name = "JournalLetter", value = JournalLetter.class)
+        @JsonSubTypes.Type(name = "JournalLetter", value = JournalLetter.class),
+        @JsonSubTypes.Type(name = "JournalLeader", value = JournalLeader.class),
```

Comment:
Should JournalLeader be here? I added it as it was missing

Reply:
I think I have put journal letter in twice, which is unsurprising as as when I think about them, they sound the same…

Reply:
It should